lang/en/tool_heartbeat.php

```diff
@@ -57,6 +57,11 @@
 $string['error_critical'] = 'CRITICAL';
 $string['error_critical_business'] = 'CRITICAL during business hours';
 $string['error_warning'] = 'WARNING';
+$string['tasklatencymonitoring'] = 'Task monitoring';
+$string['tasklatencymonitoring_desc'] = 'Enter configuration for monitoring specific cron tasks. Enter each task configuration on a new line. Configuration format is <code>\component\task\classname, (integer) runtime, (integer) starttimedrift, (integer) notrunning</code>. Runtime is the max runtime that an invididual run of a task cannot exceed. Start time drift is the drift from the scheduled run time of a task before the task begins execution. Not running is the configured time period for which a task must have run within. E.g. <code>\logstore_standard\task\cleanup_task, 5, 5, 5 </code>';
```

Comment:
I know it can be a bit of a pain, but you should move these code snippets outside of the lang string as these are not strictly "translatable" and thus should not belong in here.

Reply:
So the first block I would argue is actually translatable, the names are somewhat arbitrary, and integer may/may not be. The second one I will move out.